src/lib/cli.rs

```diff
@@ -33,8 +35,10 @@ fn run(cli_args: CliArgs, global_config: &GlobalConfig) {
         color: !cli_args.disable_color,
     });
 
-    info!("{} {}", &cli_args.command, &VERSION);
-    debug!("Written By {}", &AUTHOR);
+    if env::var("CARGO_MAKE_INTERNAL_RECURSION").unwrap_or_else(|_| "0".to_string()) != "0" {
+        info!("{} {}", &cli_args.command, &VERSION);
```

Comment:
these 2 makes a lot of sense to be printed only on the first recurresion. great idea.